chart/templates/core/storageclass.yaml

```
@@ -10,6 +10,17 @@ provisioner: csi.hetzner.cloud
 volumeBindingMode: WaitForFirstConsumer
 allowVolumeExpansion: true
 reclaimPolicy: {{ $val.reclaimPolicy | quote }}
+{{- if $val.includesRobotServers }}
+allowedTopologies:
```

```suggestion
{{- if $val.allowedTopologyCloudServer }}
allowedTopologies:
```

I find `includesRobotServers` very confusing. Setting this to `true` **excludes** Robot servers from being allowed the storage class.